repos/fdbt-site/src/pages/api/searchOperators.ts

```diff
@@ -17,25 +17,18 @@ export const removeOperatorsFromPreviouslySelectedOperators = (
     return updatedList;
 };
 
-export const addOperatorsToPreviouslySelectedOperators = (
-    rawList: string[],
-    selectedOperators: Operator[],
-): Operator[] => {
+export const addOperatorsToPreviouslySelectedOperators = (rawList: string[]): Operator[] => {
```

Comment:
Is the name of this function still accurate? It doesn't seem to be doing what the name of the function is. 

it's taking a list of strings and turning them back into Operator objects